projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.spec.ts

```diff
@@ -938,12 +941,67 @@ describe('IgxDateRangePicker', () => {
                     fixture.componentInstance.displayFormat = displayFormat;
                     fixture.detectChanges();
 
-                    expect(startInputEditor.inputFormat).toEqual(inputFormat);
+                    expect(startInputEditor.inputFormat).toEqual('MM/dd/yyyy');
```

Comment:
Erm, that's actually wrong or am I missing something? If the component has `inputFormat = 'MMM, yy'` assigned, it should not be ignored/transformed in any way